src/CommandLine.csproj

```diff
@@ -1,36 +1,46 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <!-- Import the NuSpec info about the package -->
-  <Import Project="..\pkg\nuspec.props" />
 
   <PropertyGroup>
-    <OutputType>library</OutputType>
-    <TargetFrameworks>netstandard1.5</TargetFrameworks>
-    <Version>1.4.0</Version>
-    <AssemblyVersion>1.4.0.0</AssemblyVersion>
-    <FileVersion>1.4.0.0</FileVersion>
+    <TargetFrameworks>netstandard1.5;</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net45;</TargetFrameworks>
+    <DebugType>pdbonly</DebugType>
   </PropertyGroup>
 
-  <PropertyGroup Condition="Exists('c:\Windows')">
-      <TargetFrameworks>$(TargetFrameworks);net45</TargetFrameworks>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.5|AnyCPU'">
-    <OutputPath>..\bin\Debug\</OutputPath>
+  <PropertyGroup>
+    <Version>1.5.0-beta2</Version>
+    <AssemblyVersion>1.5.0.0</AssemblyVersion>
+    <FileVersion>$(AssemblyVersion)</FileVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard1.5|AnyCPU'">
-    <OutputPath>..\bin\Release\</OutputPath>
+  <PropertyGroup>
+    <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
+    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
+    <Authors>Alex Ghiondea</Authors>
+    <Description>Parse command line arguments into user defined objects</Description>
+    <Copyright>Alex Ghiondea (c) 2017</Copyright>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/AlexGhiondea/CommandLine/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/AlexGhiondea/CommandLine</PackageProjectUrl>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>CommandLine, Command line, Command, Line, parser, objects, custom</PackageTags>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
```

Comment:
Same comments about the output path -- ideally all binaries would be in the same place.

Reply:
I feel it's better idea to add post build script to copy assemblies around than overriding default output path, and since nuget is generated and included as artifact in CI, why do ppl need dlls

Reply:
Answered this above.